cmd/swapcli/main.go

```diff
@@ -519,35 +549,55 @@ func runMake(ctx *cli.Context) error {
 		return err
 	}
 
+	ethAssetStr := ctx.String("eth-asset")
```

Comment:
This just got moved around. We should update that to use a constant. It might be nice if the flag passed to `balances` (currently, `--token`) matched this flag to `make`. I'm not sure `--eth-asset` is an intuitive flag name for `balances` though. What would you think about using `--token` for both?

Reply:
i think `--token` for this flag would make sense as well, might be more intuitive actually. if it's not set then we're just using eth, otherwise a token.

Reply:
Done. We are now using `--token` for `make` and `balances`.